programs/perpetuals/tests/native/main.rs

```diff
@@ -29,6 +29,9 @@ pub async fn test_integration() {
     tests_suite::staking::liquid_staking_overlap_remove_same_as_overlap().await;
     tests_suite::staking::liquid_staking_overlap_remove_more_than_overlap().await;
 
+    tests_suite::vesting::vote().await;
```

Comment:
This test suite wasn't used, I think it's a mistake? Seems it doesn't work though, so I'll comment it for now (unrelated to this PR)